db/academyPortal_2018-11-26.sql

```diff
@@ -20,6 +20,53 @@
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
 
+# Dump of table companySizeLink
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `companySizeLink`;
+
+CREATE TABLE `companySizeLink` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `size` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `companySizeLink` WRITE;
+/*!40000 ALTER TABLE `companySizeLink` DISABLE KEYS */;
+
+INSERT INTO `companySizeLink` (`id`, `size`)
+VALUES
+	(1,'<5'),
+	(2,'5-30'),
+	(3,'31-60'),
+	(4,'61-100'),
+	(5,'101-250'),
+	(6,'>250');
+
+/*!40000 ALTER TABLE `companySizeLink` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table hiringPartners
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `hiringPartners`;
+
+CREATE TABLE `hiringPartners` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `companyName` varchar(255) NOT NULL DEFAULT '',
+  `size` int(11) unsigned NOT NULL,
+  `techStack` text NOT NULL,
+  `postcode` varchar(255) NOT NULL DEFAULT '',
```

Comment:
I don't think there's any postcode longer than 7 characters, 8 if you're including the space.

Reply:
Fixed in abf06b6